src/baseline_system/parsers/props_wrapper.py

```diff
@@ -188,6 +192,15 @@ def create_implicit_proposition(self, *arg_symbols):
                 "Arguments": arg_symbols
             }
 
+    # retrieve spacy's named entities, only those with relevant label
+    def _get_named_entities(self):
+        relevant_labels = ["PERSON", "NORP", "FACILITY", "ORG", "GPE", "LOC",
+                           "PRODUCT", "EVENT", "WORK_OF_ART", "LANGUAGE",
+                           "DATE", "TIME", "QUANTITY", "MONEY"]
```

Comment:
I think it would be better to declare constants out of the scope of functions, and instead in the class level, and in capital letters. That way it's easier to spot and manage them over time.
Even better - does spacy have constants for these (e.g., ```spacy.PERSON``` or something similar) ?
that would be best IMO, since any update to spacy won't incur any code breaks for us.

Reply:
since these are labels, I think it doesn't quite make sense to have constants for them. spacy dont use constants, because it is the same as using string labels - no one will want to change the label string without changing the meaning of the categorization.